### PR TITLE
Setup local dev for insights-ingress-go.

### PR DESCRIPTION
**Pull request by myersCody**
~Blocked by: https://github.com/RedHatInsights/insights-ingress-go/pull/112~

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -196,9 +196,9 @@ local-dev-down:
 	sudo lsof -t -i tcp:8081 | xargs kill -9
```
Comment (abaiken) [inline Makefile]: `make local-dev-down` does not work.

Comment (myersCody) [reply Makefile]: I have updated the path to point at `insights-ingress-go`. Great Catch ash 👍

**Comment by abaiken** [comment]
I ran into the following issue: 
```
ERROR: Service 'ingress' failed to build: oci runtime error: container_linux.go:265: starting container process caused "exec: \"/bin/sh\": stat /bin/sh: no such file or directory"
```
but resolved it by following the instructions [here](https://github.com/docker/for-mac/issues/1396).

**Comment by abaiken** [comment]
Everything worked 🎉and looks good. The only question that I have is if we shoul still refer to ingress as the file upload service everywhere. Do we want to update documentation to say ingress service (since the file-upload service was a thing of the past) or are we okay since it is technically the thing you upload files to? Thoughts @kholdaway  @myersCody?

**Comment by myersCody** [comment]
> I ran into the following issue:
> 
> ```
> ERROR: Service 'ingress' failed to build: oci runtime error: container_linux.go:265: starting container process caused "exec: \"/bin/sh\": stat /bin/sh: no such file or directory"
> ```
> 
> but resolved it by following the instructions [here](https://github.com/docker/for-mac/issues/1396).

Great work debugging this issue @abaiken

**Comment by myersCody** [comment]
> Everything worked 🎉and looks good. The only question that I have is if we shoul still refer to ingress as the file upload service everywhere. Do we want to update documentation to say ingress service (since the file-upload service was a thing of the past) or are we okay since it is technically the thing you upload files to? Thoughts @kholdaway @myersCody?

I am okay with changing it if we think it is necessary. I think whatever is the least confusing for people on-boarding to the project is the direction we should go with. However, considering that I am relatively new to the project, I may not be the best source to decide which direction we should go in. 

🤷‍♂

File: README.md [hunk 2]
```diff
@@ -267,7 +266,7 @@ make upload-data file=<path/to/your-data.tar.gz>
 
 You need to replace `<path/to/your-data.tar.gz>` with either the absolute or relative path to the `tar.gz` file that you want to upload to Insights.
 
-After running this command if you see `HTTP 202` like the following lines in your output logs, it means your file upload to Insights was successful:
+After running this command if you see `HTTP 202` like the following lines in your output logs, it means your ingress to Insights was successful:
```
Comment (abaiken) [inline README.md]: This doesn't make sense. it should be `upload to the ingress service`

**Pull request merged**